### PR TITLE
fix: add region tags to Cloud Run config sample

### PR DESCRIPTION
**Pull request by ace-n**
```release-note:none
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 1]
```diff
@@ -35,7 +40,10 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
           name  = "baz"
           value = "quux"
         }
+        # [END cloudrun_service_configuration_env_var]
 
+        # [START cloudrun_service_configuration_limit_memory]
+        # [START cloudrun_service_configuration_limit_cpu]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:46]: Does enclosure order matter ?

`cloudrun_service_configuration_limit_memory` seems to contain `cloudrun_service_configuration_limit_cpu` but ends before `cloudrun_service_configuration_limit_memory`

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 2]
```diff
@@ -47,18 +55,34 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
             memory = "512Mi"
           }
         }
+        # [END cloudrun_service_configuration_limit_memory]
+        # [END cloudrun_service_configuration_limit_cpu]
+
+        # [START cloudrun_service_configuration_containers]
       }
+      # [END cloudrun_service_configuration_containers]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:63]: this enclosure only contains a single bracket. Is that intentional?

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 3]
```diff
@@ -47,18 +55,34 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
...
 
+      # [START cloudrun_service_configuration_concurrency]
       # Maximum concurrent requests
       # https://cloud.google.com/run/docs/configuring/concurrency
       container_concurrency = 80
+      # [END cloudrun_service_configuration_concurrency]
+
+      # [START cloudrun_service_configuration_containers]
     }
+    # [END cloudrun_service_configuration_containers]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:79]: enclosure contains a single bracket

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 4]
```diff
@@ -47,18 +55,34 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
...
 
+    # [START cloudrun_service_configuration_max_instances]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:81]: This block id starts multiple times. Is that intentional.

See lines `81` and `110`

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 5]
```diff
@@ -47,18 +55,34 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
...
+    # [START cloudrun_service_configuration_min_instances]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:82]: This block id starts multiple times. Is that intentional?

See lines `82` and `111`

**Review by ScottSuarez — COMMENTED** [review]
Questions with how the layout and tags are setup.

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 1 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:46]: Nope!

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 2 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:63]: Yep - enclosures with the same tag-name will be concatenated together.

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 3 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:79]: Ditto:

> Enclosures with the same tag-name will be concatenated together.

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 4 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:81]: Nope - fixed.

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 5 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:82]: Ditto - fixed; thanks for catching. 🙂

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 6]
```diff
@@ -73,15 +97,23 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
...
+      # [END cloudrun_service_configuration_min_instances]
 
+      # [START cloudrun_service_configuration_labels]
       # Labels
       # https://cloud.google.com/run/docs/configuring/labels
       labels = {
         foo : "bar"
         baz : "quux"
       }
+      # [END cloudrun_service_configuration_max_instances]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:110]: this tag starts on line`81` but ends on line `100`, `110`, and `115`.. probably need a single end tag or multiple starts

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb [hunk 7]
```diff
@@ -73,15 +97,23 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
...
+      # [END cloudrun_service_configuration_min_instances]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:111]: This tag starts on line `82` and ends online `111` and `116`.

**Review by ScottSuarez — COMMENTED** [review]
Some more concerns I see with block start and ends not matching !

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 6 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:110]: Thanks! Fixed.

File: mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb (hunk 7 above)
Comment (ace-n) [reply mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb:111]: Thanks! Fixed.

**Review by ScottSuarez — APPROVED** [review]
Thanks for the work!

**Pull request merged**